src/index.ts

```diff
@@ -113,32 +113,33 @@ export = (app: Application) => {
         }
       }
     `, {
-      owner: owner,
-      repo: repo,
-      refQualifiedName: `refs/heads/${branchName}`
+      owner,
+      repo
     }, {})
     if (!result.data) { return [] }
-    return result.data.repository.ref.associatedPullRequests.nodes.map((node: any) => ({
-      number: node.number,
-      repo: node.repository.name,
-      owner: node.repository.owner.login
-    }))
+    const allPullRequests = result.data.repository.pullRequests.edges.map((edge: any) => {
+      const node = edge.node
+      return {
+        number: node.number,
+        commitId: node.headRefOid,
+        repo: node.repository.name,
+        owner: node.repository.owner.login
+      }
+    })
+
+    return allPullRequests.filter((pr: any) => pr.commitId === commitId)
```

Comment:
```suggestion
    return result.data.repository.pullRequests.edges
        .filter(({ node }: any) => node.headRefOid === headRefOid)
        .map(({ node }: any) => ({
            number: node.number,
            repo: node.repository.name,
            owner: node.repository.owner.login
        });
```
We do not need to expose `commitId`/`headRefOid`, as it is part of the input. We can filter and map in one go.